### PR TITLE
Updated the text for Expand/Collapse section to reflect new search ta…

### PR DESCRIPTION
**Pull request by hganapathy**
- Updated the text for Expand/Collapse section to reflect new search table options
- Updated image to reflect the same on both Expand/Collapse section and Copy message section

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 1]
```diff
@@ -6,21 +6,21 @@ description: When you run a search query, messages display in the Message, Aggre
 
 
 
-When you run a search your results are provided in a **Messages** tab. If the search conducted an aggregation you'll also get an **Aggregates** tab.
+When you run a search your results are provided in a **Messages** tab. If the search conducted an aggregation you'll also get an **Aggregates** tab.
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:9]:
```suggestion
When you run a search, the results are displayed in the **Messages** tab. If the search conducted an aggregation, an **Aggregates** tab will appear.
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 2]
```diff
@@ -6,21 +6,21 @@ description: When you run a search query, messages display in the Message, Aggre
...
 
 ![Messages and Aggregates tab.png](/img/search/get-started-search/search-page/messages-aggregates-tab.png)
 
-The **Signatures** tab is shown when using [LogReduce](/docs/search/logreduce).
+The **Signatures** tab is shown when using [LogReduce](/docs/search/logreduce).
 
 ![Signatures tab.png](/img/search/get-started-search/search-page/signatures-tab.png)
 
 ## Column adjustments
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:17]:
```suggestion
## Column Adjustments
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 3]
```diff
@@ -6,21 +6,21 @@ description: When you run a search query, messages display in the Message, Aggre
...
 
-The table columns can be modified by the following:
+The table columns can be modified by the following:
 
-* **Move a Column.** Hold click and drag a column from the header row to a different location in the table.
-* **Change column width.** Hold click and drag the vertical line in between the columns to adjust the width. <br/>![resize column.png](/img/search/get-started-search/search-page/resize-column.png)
-* **Change column height.** Double click the vertical line to the right of a column name to automatically resize the width to fit the content.
+* **Move a Column.** Hold click and drag a column from the header row to a different location in the table.
+* **Change column width.** Hold click and drag the vertical line in between the columns to adjust the width. <br/>![resize column.png](/img/search/get-started-search/search-page/resize-column.png)
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:22]:
```suggestion
* **Change column width**. You can adjust the width of the columns by holding down the click and dragging the vertical line in between them. <br/>![resize column.png](/img/search/get-started-search/search-page/resize-column.png)
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 4]
```diff
@@ -6,21 +6,21 @@ description: When you run a search query, messages display in the Message, Aggre
...
 
 ## Column adjustments
 
-The table columns can be modified by the following:
+The table columns can be modified by the following:
 
-* **Move a Column.** Hold click and drag a column from the header row to a different location in the table.
-* **Change column width.** Hold click and drag the vertical line in between the columns to adjust the width. <br/>![resize column.png](/img/search/get-started-search/search-page/resize-column.png)
-* **Change column height.** Double click the vertical line to the right of a column name to automatically resize the width to fit the content.
+* **Move a Column.** Hold click and drag a column from the header row to a different location in the table.
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:21]:
```suggestion
* **Move a Column.** To move a column, click and hold down your mouse button on the column, then drag it to a new location.
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 5]
```diff
@@ -6,21 +6,21 @@ description: When you run a search query, messages display in the Message, Aggre
...
+* **Change column width.** Hold click and drag the vertical line in between the columns to adjust the width. <br/>![resize column.png](/img/search/get-started-search/search-page/resize-column.png)
+* **Change column height.** Double click the vertical line to the right of a column name to automatically resize the width to fit the content.
 * **Pin columns**: You can pin columns in the table, either in the **Messages** tab or the **Aggregates** tab. When a column is pinned you can scroll to the right and still view it. To pin a column, hover over the column header, and click the pin icon that appears.
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:24]:
```suggestion
* **Pin columns**: You can pin columns in the table, either in the **Messages** tab or the **Aggregates** tab. When a column is pinned, you can scroll to the right and still view it. To pin a column, hover over the column header, and click the pin icon that appears.
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 6]
```diff
@@ -30,30 +30,30 @@ After you’ve pinned a column, the pin icon has a slash through it. You can cli
 
 ## Expand and collapse
 
-By default, the **Messages** tab shows the collapsed version of messages, where the row has a height limit of 500 pixels. Click **Expand All** on the upper right side of the **Messages** table to view the full contents of all your log messages.
+By default, the **Messages** tab shows the collapsed version of messages with each row showing up to 10 lines. Click **Expand/Collapse** on the upper right side of the **Messages** table to select  **Expand All Rows** or  **Expand All Rows And JSON** option view the full contents of all your log messages including JSON logs.
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:33]:
```suggestion
By default, the **Messages** tab shows the collapsed version of messages with each row showing up to 10 lines. To see the full contents of all log messages, including JSON logs, click the **Expand/Collapse** button on the upper right side of the **Messages** table and choose either **Expand All Rows** or **Expand All Rows And JSON** option.
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 7]
```diff
@@ -30,30 +30,30 @@ After you’ve pinned a column, the pin icon has a slash through it. You can cli
 
 ## Expand and collapse
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:31]:
```suggestion
## Expand and Collapse
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 8]
```diff
@@ -30,30 +30,30 @@ After you’ve pinned a column, the pin icon has a slash through it. You can cli
...
 
-By default, the **Messages** tab shows the collapsed version of messages, where the row has a height limit of 500 pixels. Click **Expand All** on the upper right side of the **Messages** table to view the full contents of all your log messages.
+By default, the **Messages** tab shows the collapsed version of messages with each row showing up to 10 lines. Click **Expand/Collapse** on the upper right side of the **Messages** table to select  **Expand All Rows** or  **Expand All Rows And JSON** option view the full contents of all your log messages including JSON logs.
 
 ![expand message option.png](/img/search/get-started-search/search-page/expand-message-option.png)
 
-If expanded, the option changes to **Collapse All**. Click it to switch back the initial collapsed view.
+Click  **Expand/Collapse** on the upper right side of the **Messages** table and select  **Collapse All Rows**  to switch back the initial collapsed view.
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:37]:
```suggestion
Click  **Expand/Collapse** on the upper right side of the **Messages** table and select  **Collapse All Rows**  to switch back to the initial collapsed view.
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 9]
```diff
@@ -30,30 +30,30 @@ After you’ve pinned a column, the pin icon has a slash through it. You can cli
...
 
 ## Copy message
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:39]:
```suggestion
## Copy Message
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 10]
```diff
@@ -30,30 +30,30 @@ After you’ve pinned a column, the pin icon has a slash through it. You can cli
...
 
-The **Messages** tab allows you to right-click a table cell to copy the raw message to your clipboard.
+The **Messages** tab allows you to right-click a table cell to copy the raw message to your clipboard.
 
 ![copy message option.png](/img/search/get-started-search/search-page/copy-message-option.png)
 
 ## Keyboard accessibility
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:45]:
```suggestion
## Keyboard Accessibility
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 11]
```diff
@@ -30,30 +30,30 @@ After you’ve pinned a column, the pin icon has a slash through it. You can cli
...
 
-Click anywhere on the table in the **Messages** or **Aggregates** tab and you can use the up and down keyboard keys to navigate your results.
+Click anywhere on the table in the **Messages** or **Aggregates** tab and you can use the up and down keyboard keys to navigate your results.
 
 ## Pages
 
 If you have many pages of results, you have several options for navigation:
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:52]:
```suggestion
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Some formatting changes.

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 12]
```diff
@@ -6,54 +6,53 @@ description: When you run a search query, messages display in the Message, Aggre
...
-## Expand and collapse
+## Expand and Collapse
 
-By default, the **Messages** tab shows the collapsed version of messages, where the row has a height limit of 500 pixels. Click **Expand All** on the upper right side of the **Messages** table to view the full contents of all your log messages.
+By default, the **Messages** tab shows the collapsed version of messages with each row showing up to 10 lines. To see the full contents of all log messages, including JSON logs, click the **Expand/Collapse** button on the upper right side of the **Messages** table and choose either **Expand All Rows** or **Expand All Rows And JSON** option.
 
 ![expand message option.png](/img/search/get-started-search/search-page/expand-message-option.png)
 
-If expanded, the option changes to **Collapse All**. Click it to switch back the initial collapsed view.
+Click  **Expand/Collapse** on the upper right side of the **Messages** table and select  **Collapse All Rows**  to switch back to the initial collapsed view.
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:37]:
```suggestion
Click **Expand/Collapse** on the upper right side of the **Messages** table, then select  **Collapse All Rows**  to switch back to the initial collapsed view.
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 13]
```diff
@@ -6,54 +6,53 @@ description: When you run a search query, messages display in the Message, Aggre
...
 
-## Copy message
+## Copy Message
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:39]: Reverting edit. Our style is to use Sentence Case (see [Style Guide](https://help.sumologic.com/docs/contributing/style-guide/#capitalization)).

```suggestion
## Copy message
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 14]
```diff
@@ -6,54 +6,53 @@ description: When you run a search query, messages display in the Message, Aggre
...
 
-The **Messages** tab allows you to right-click a table cell to copy the raw message to your clipboard.
+The **Messages** tab allows you to right-click a table cell to copy the raw message to your clipboard.
 
 ![copy message option.png](/img/search/get-started-search/search-page/copy-message-option.png)
 
-## Keyboard accessibility
+## Keyboard Accessibility
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:45]: Reverting edit. Our style is to use Sentence Case (see [Style Guide](https://help.sumologic.com/docs/contributing/style-guide/#capitalization)).

```suggestion
## Keyboard accessibility
```

File: docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md [hunk 15]
```diff
@@ -6,54 +6,53 @@ description: When you run a search query, messages display in the Message, Aggre
...
 
 ![Messages and Aggregates tab.png](/img/search/get-started-search/search-page/messages-aggregates-tab.png)
 
-The **Signatures** tab is shown when using [LogReduce](/docs/search/logreduce).
+The **Signatures** tab is shown when using [LogReduce](/docs/search/logreduce).
 
 ![Signatures tab.png](/img/search/get-started-search/search-page/signatures-tab.png)
 
-## Column adjustments
+## Column Adjustments
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md:17]: Reverting edit. Our style is to use Sentence Case (see [Style Guide](https://help.sumologic.com/docs/contributing/style-guide/#capitalization)).

```suggestion
## Column adjustments
```

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
@hganapathy , thanks for your edits! It looks like something became corrupted with the PR. I've moved your changes to a new PR to ensure nothing broken gets merged: https://github.com/SumoLogic/sumologic-documentation/pull/1879.

**Pull request closed without merging**